vassal-app/src/main/java/VASSAL/counters/BorderOutline.java

```diff
@@ -327,4 +329,13 @@ private static class LogicalCompareConfigurer extends TranslatingStringEnumConfi
       super(null, null, LogicalCompareMode.getSymbols(), LogicalCompareMode.getKeys());
     }
   }
+
+  /**
+   * {@link SearchTarget}
+   * @return a list of any Property Names referenced in the Decorator, if any (for search)
+   */
+  @Override
+  public List<String> getPropertyList() {
+    return Arrays.asList(propertyName, propertyName2);
+  }
```

Comment:
Tip: Since you already have `List` imported, you can use `List.of()` and not need to import `Arrays` additionally.

Reply:
All fixed. I'll admit I can't keep all the different ways to assemble List/Array/Collection things in my head.